.github/workflows/airgap.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Airgapped build
+on:
+  pull_request:
+
+jobs:
+  airgapped_build:
+    name: Airgapped build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install system dependencies
+        run: grep '^[^#]' apt-requirements.txt | xargs sudo apt install -y
```

Comment:
Maybe we should extend `ci/install-package-dependencies.sh` instead?

Reply:
Possibly, but that script is way more complicated than what we need (this one line).

There are so many scripts in `ci/` that are unreadable and unmaintainable. I'd like to cut these down or delete most of them entirely in the move to GHA.